### ISSUE TITLE
[Bug]: TaskItem - Clicking checkbox invokes onBlur on the editor

### ISSUE THREAD

**Issue by jarlef** [ueberdosis/tiptap#5519]
### Affected Packages

extension-task-item

### Version(s)

2.6.4

### Bug Description

If the user adds a task list and toggles the checkbox the onBlur on the editor is invoked which annoying if you use this event to persist data.

```ts
const extensions = [
  StarterKit,
  TaskList,
  TaskItem.configure({
    nested: true,
  }),
];
```
```tsx
const editor = useEditor({
    extensions,
    content: value,
    autofocus: true,
  });

const handleBlur = () => {
  console.log('blur got invoked')
};

return <EditorContent editor={editor} onBlur={() => handleBlur()} />;
```

### Browser Used

Chrome

### Expected Behavior

When toggling the checkbox the onBlur should not be invoked

### Dependency Updates

- [X] Yes, I've updated all my dependencies.

**Comment by nperez0111** [comment]
@jarlef would be happy to accept a PR on this. Probably just need to have some sort of event handler on the checkbox https://github.com/ueberdosis/tiptap/blob/6543f059ffe0f2cae4b434acd6672f991052edd8/packages/extension-task-item/src/task-item.ts#L140

**Comment by jarlef** [comment]
@nperez0111 I have pushed a fix for this.

**Comment by nperez0111** [comment]
This has been released with 2.7.0-pre.0 and will be released in tiptap 2.7.0 when this beta has been tested by more users
